include/geometric_shapes/bodies.h

```diff
@@ -499,21 +499,21 @@ class BodyVector
   BodyVector();
 
   /** \brief Construct a body vector from a vector of shapes, a vector of poses and a padding */
-  BodyVector(const std::vector<shapes::Shape*>& shapes, const EigenSTL::vector_Isometry3d& poses, double padding = 0.0);
+  BodyVector(const std::vector<shapes::Shape*>& shapes, const EigenSTL::vector_Affine3d& poses, double padding = 0.0);
```

Comment:
I understand that this is currently blocking you in your porting efforts, but it seems more efficient and forward thinking to me to assume that the `Isometry3d` migration will happen in all ROS pkgs that use `Eigen::Affine3d`.

Instead of reverting all those changes in MoveIt and related pkgs, perhaps forward-porting the `Isometry3d` changes to the pkgs that *should* have it would be more efficient in the long run.

Reply:
+1 we do not want to revert back to affine3d